adapters/mqtt-vertx/src/main/java/org/eclipse/hono/adapter/mqtt/impl/MappedMessage.java

```diff
@@ -0,0 +1,17 @@
+package org.eclipse.hono.adapter.mqtt.impl;
+
+import io.vertx.mqtt.messages.MqttPublishMessage;
+import org.eclipse.hono.adapter.mqtt.MqttContext;
+import org.eclipse.hono.util.ResourceIdentifier;
+
+public class MappedMessage {
```

Comment:
This seems to be a class that is only used within this package so I propose to declare it and its properties package private in order to prevent other code to start using an depending on it ...